src/shogun/regression/LinearRidgeRegression.cpp

```diff
@@ -75,24 +77,26 @@ bool CLinearRidgeRegression::train_machine(CFeatures* data)
 	for (int32_t i=0; i<num_feat; i++)
 		kernel_matrix.matrix[i+i*num_feat]+=m_tau;
 
+	// Get vectors and matrices in eigen form
```

Comment:
comment unnecessary as the code totally explains it. Only write comments if they are helpful